client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -675,6 +742,27 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: uploadIcon,
 		},
+		{
+			name: 'manageSettingsGeneral',
+			label: __( 'Manage general settings' ),
+			context: [ '/settings' ],
+			callback: setStateCallback(
+				'manageSettingsGeneral',
+				__( 'Select site to manage general settings' )
+			),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					const link =
+						( site.jetpack && ! site.is_wpcom_atomic ) ||
+						'wp-admin' === site.options?.wpcom_admin_interface
```

Comment:
Would it make sense to add an issue to abstract those?

Reply:
- Created https://github.com/Automattic/dotcom-forge/issues/4764